docs/NNNN-schema-registry.md

```diff
@@ -0,0 +1,152 @@
+# OCFL Community Extension - Schema Registry
+
+Extension Name: NNNN-schema-registry
+Authors: P. Cornwell, D. Granville
+Minimum OCFL Version: 1.0
+OCFL Community Extensions Version: 1.0
+Extension Version: 0.1
+Obsoletes: n/a
+Obsoleted by: n/a
+
+## Overview
+An OCFL object will typically contain metadata serialised in JSON or XML files. These will normally conform to one or more specified schemata and reference the external JSONSchema/XSD/DTD.
+
+In order for an OCFL root to represent a self-consistent repository, the specific versions of each schema referenced by content must be available. Maintaining a local copy of the schema is therefore prudent, especially where longer-term preservation is a goal and external online references cannot be relied upon.
+
+This extension stores a copy of each schema referenced by the content of OCFL objects, thus avoiding the need to store a copy of the schema redudantly in each OCFL object. It also provides a convenient reference point for archive management software and future users inspecting the contents of the OCFL object.
+
+We describe a standardised layout for a schema repository - essentially implemented as a 'meta' OCFL object: a sub-directory containing local copies of schemata. This directory contains all schemata, with standard OCFL integrity checks, in use within the OCFL root.
+
+## Parameters
```

Comment:
Can you describe the parameters here in the format [outlined here](https://github.com/OCFL/extensions/#parameters)?

Reply:
Done in PR below.